### PR TITLE
Add links to eBPFChirp newsletter tutorials and guides

### PR DESCRIPTION
**Pull request by dorkamotorka**
@qmonnet Feel free to suggest any changes I should apply.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -183,6 +184,12 @@ If you are new to eBPF, you may want to try the links described as "introduction
 - [A Deep Dive into eBPF: Writing an Efficient DNS Monitoring.](https://medium.com/@nurkholish.halim/a-deep-dive-into-ebpf-writing-an-efficient-dns-monitoring-2c9dea92abdf) - A detailed explanation of methods used to capture DNS requests at the socket filter layer.
 - [eBPF Developer Tutorial - Learn eBPF by examples](https://eunomia.dev/tutorials/) - Start with eBPF basics and progress to advanced topics using 20+ hands-on tutorials and examples. Covers performance, networking, and security with libbpf and CO-RE. Available in Chinese and English.
 - [Catch Performance Regressions in eBPF](https://bencher.dev/docs/explanation/talks/#linuxcon-2023-12-may-23) - A step-by-step guide to benchmarking both the client and kernel eBPF code written in Rust.
+- [Loops and Iterators in eBPF](https://cloudchirp.substack.com/p/loops-and-iterators-in-ebpf?r=1m86sa) - Newsletter about all the ways to loop and iterate in eBPF.
```
Comment (qmonnet) [inline readme.md:187]: Hi, thanks and apologies for the delay.

The contents look good (nice work!), so I'm fine with adding the links. That's a lot of links, but then that's also a lot of interesting reads.

Could you please clean up the links? Some seem to have a parameter that is not strictly necessary to access the resource:

```suggestion
- [Loops and Iterators in eBPF](https://cloudchirp.substack.com/p/loops-and-iterators-in-ebpf) - Newsletter about all the ways to loop and iterate in eBPF.
```

**Review by qmonnet — CHANGES_REQUESTED** [review]

File: readme.md (hunk 1 above)
Comment (dorkamotorka) [reply readme.md]: Oh, I see - this should now be resolved.

File: readme.md [hunk 2]
```diff
@@ -183,6 +184,13 @@ If you are new to eBPF, you may want to try the links described as "introduction
...
+- [What Insights Can eBPF Provide into Real-Time SSL/TLS Encrypted Traffic and How?](https://cloudchirp.substack.com/p/what-insights-can-ebpf-provide-into) - A step-by-step guide how eBPF can observe encrypted network traffic.
+- [Can eBPF Detect Redis Message Patterns Before They Become Problems?](https://cloudchirp.substack.com/p/can-ebpf-detect-redis-message-patterns) - A step-by-step guide how eBPF can observe Redis communication between client and server.
+- [Transparent Proxy Implementation using eBPF and Go](https://cloudchirp.substack.com/p/transparent-proxy-implementation) - A step-by-step guide on how to implement a transparent proxy using eBPF.
+- [eBPF-Powered Load Balancing](https://cloudchirp.substack.com/p/ebpf-powered-load-balancing-for-so_reuseport) - Learn how eBPF can infer custom load-balancing for services listening on the same port, through the SO_REUSEPORT TCP option.
+- [Unit Testing eBPF Programs](https://ebpfchirp.substack.com/p/unit-testing-ebpf-programs) - Learh how you can unit test your eBPF programs using libbpf.
```
Comment (qmonnet) [inline readme.md:192]:
```suggestion
- [Unit Testing eBPF Programs](https://ebpfchirp.substack.com/p/unit-testing-ebpf-programs) - Learn how you can unit test your eBPF programs using libbpf.
```

File: readme.md [hunk 3]
```diff
@@ -370,6 +378,9 @@ If you are new to eBPF, you may want to try the links described as "introduction
 - [ebpfkit-monitor](https://github.com/Gui774ume/ebpfkit-monitor) - An utility to statically analyze eBPF bytecode or monitor suspicious eBPF activity at runtime. It was specifically designed to detect ebpfkit.
 - [Bad BPF](https://github.com/pathtofile/bad-bpf) - A collection of malicious eBPF programs that make use of eBPF's ability to read and write user data in between the usermode program and the kernel.
 - [TripleCross](https://github.com/h3xduck/TripleCross) - A Linux eBPF rootkit with a backdoor, C2, library injection, execution hijacking, persistence and stealth capabilities.
+- [Securing Kubernetes Workloads using LSM-BPF](https://ebpfchirp.substack.com/p/securing-kubernetes-workloads-using) - Learn how tools like KubeArmor leverage LSM-BPF to prevent malicious attacks such as Ransomware or Cryptojacking.
+- [Protecting HashiCorp Vault using KubeArmor](https://ebpfchirp.substack.com/p/protecting-k8s-secrets-using-kubearmor) - Learn how KubeArmor blocks malicious attacks on HashiCorp Vault and why use it instead of solutions like Tetragon or Falco.
```
Comment (qmonnet) [inline readme.md:382]: I have reservations about this link, and in particular about pushing KubeArmor over Tetragon for the wrong reasons:

> Tetragon, while capable of blocking threats, only does so after malicious actions have already occurred.

This is not correct. Tetragon blocks the syscall before it succeeds. I invite you to get in touch with the Tetragon folks for more details. See also https://tetragon.io/docs/concepts/enforcement/#override-return-value

(Disclaimer: I used to work at Isovalent)

My suggestion would be to remove this link from your PR for now, merge the rest of the PR, and we can discuss adding the link again after this point is fixed.

**Review by qmonnet — CHANGES_REQUESTED** [review]

**Review by qmonnet — APPROVED** [review]
The other KubeArmor links were fine, but OK, that works for me. Thanks!

**Pull request merged**